app/js/tests/view_handlers/test_ppl_recurrent_warning.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+var test = require( 'tape' ),
+	sinon = require( 'sinon' ),
+	pplRecurrentWarning = require( '../../lib/view_handler/ppl_recurrent_warning' ),
+	createPPLRecurrentWarningHandler = pplRecurrentWarning.createPPLRecurrentWarningHandler
+	;
+
+function createElement() {
+	return {
+		show: sinon.spy(),
+		hide: sinon.spy()
+	};
+}
+
+test( 'When interval is recurring, warning is shown', function ( t ) {
+	var element = createElement(),
+		formValues = {
+			paymentType: 'PPL',
+			paymentIntervalInMonths: 12
+	    },
+		handler = createPPLRecurrentWarningHandler( element );
+
+	handler.update( formValues );
```

Comment:
Why did you create a local var for `formValues`? You can just pass it in here no? More local...